### PR TITLE
Lv2libs

### PR DESCRIPTION
**Pull request by magnetophon**
###### Motivation for this change

Update lv2 and related libs.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] OS X
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by magnetophon** [comment]
It [seems](https://travis-ci.org/NixOS/nixpkgs/jobs/172235757#L3961) some related packages need to be updated for travis to be OK.
Should I make a single PR out of this and [Lv2plugins](https://github.com/NixOS/nixpkgs/pull/20045)?

File: pkgs/development/libraries/audio/lv2/lv2-git.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  src = fetchgit {
+    url = "http://lv2plug.in/git/cgit.cgi/lv2.git";
+    rev = "b36868f3b96a436961c0c51b5b2dd71d05da9b12";
+    sha256 = "1sx39j0gary2nayzv7xgqcra7z1rcw9hrafkji05aksdwf7q0pdm";
+  };
+
+  buildInputs = [ gtk2 libsndfile pkgconfig python ];
+
+  configurePhase = "python waf configure --prefix=$out";
```
Comment (FRidh) [inline pkgs/development/libraries/audio/lv2/lv2-git.nix]: ${python.interpreter}

File: pkgs/development/libraries/audio/raul/default.nix [hunk 2]
```diff
@@ -1,13 +1,13 @@
-{ stdenv, fetchsvn, boost, gtk2, pkgconfig, python }:
+{ stdenv, fetchgit, boost, gtk2, pkgconfig, python }:
 
 stdenv.mkDerivation rec {
-  name = "raul-svn-${rev}";
-  rev = "5675";
+  name = "raul-git-${rev}";
```
Comment (FRidh) [inline pkgs/development/libraries/audio/raul/default.nix]: unstable

File: pkgs/development/libraries/audio/lv2/lv2-git.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, gtk2, libsndfile, pkgconfig, python }:
```
Comment (FRidh) [inline pkgs/development/libraries/audio/lv2/lv2-git.nix]: the filename should also be `lv2-unstable.nix`

**Comment by magnetophon** [comment]
@FRidh Done, sorry for the sloppiness.

**Comment by magnetophon** [comment]
@FRidh travis says: ``The log length has exceeded the limit of 4 Megabytes``, but looks fine otherwise.
I don't understand why it's such a big compile. Why is ghc in there for example?
Should this be in staging?

**Comment by FRidh** [comment]
This looks fine, it just needs to be rebased. You can ignore Travis. Travis runs `nox-review`, either your packages require ghc in some dependency, or ghc is affected by these changes.

**Comment by magnetophon** [comment]
@FRidh :  I was away for a whil, but finally managed to rebase.

**Comment by FRidh** [comment]
I didn't check the individual commits and thus missed 5abcc7d
Mass replacing changes in a single commit is something to avoid, because it makes cherry-picking a lot more difficult. Furthermore, it touches more packages than were mentioned in the commit message. E.g., `ingen` did not evaluate anymore.

**Comment by magnetophon** [comment]
@FRidh Than,ks for fixing and merging!

**Pull request merged**